### PR TITLE
Show error message if unable to load alphabet file

### PR DESCRIPTION
**Pull request by dabinat**
Addresses #2186.

### CODE REVIEW

File: native_client/alphabet.h [hunk 1]
```diff
@@ -17,6 +17,12 @@ class Alphabet {
 public:
   Alphabet(const char *config_file) {
     std::ifstream in(config_file, std::ios::in);
+
+    if (!in) {
+      std::cerr << "Unable to load alphabet file " << config_file << std::endl;
+      abort();
```
Comment (lissyx) [inline native_client/alphabet.h]: Likely we would prefer a proper error code and handling by the caller?

Comment (dabinat) [reply native_client/alphabet.h]: Ok, but alphabet.h might need redesigning then. Maybe it's doing too much in the constructor and most of that code should be moved to another function?

e.g.

````
Alphabet *alpha = new Alphabet();
alpha->loadFile(filename);
````

Comment (lissyx) [reply native_client/alphabet.h]: @dabinat Maybe yes. What's your take @reuben ?

Comment (reuben) [reply native_client/alphabet.h]: Yeah, I had to do the same with the ModelState implementations, moving things to a separate initialization method. Sounds good to me.

Comment (lissyx) [reply native_client/alphabet.h]: @dabinat Do you think you can address that?

Comment (dabinat) [reply native_client/alphabet.h]: I’d assumed when I posted the PR it was essentially an easy 1-line fix and I haven’t had time to dive deeper yet. I will be able to later this week though.

**Pull request closed without merging**